### PR TITLE
Ignore *.secret

### PR DESCRIPTION
**Pull request by yet-another-account**
Repost of #182 , now with the lower threshold

### CODE REVIEW

**Comment by yet-another-account** [comment]
Huh? this has no conflicts with base at all :/

**Comment by phil-r** [comment]
@eukaryote31 Chaos...

**Comment by ECrownofFire** [comment]
We should probably remove old labels...

**Comment by yet-another-account** [comment]
Wait, why does chaos tag PRs as `conflicts` if they don't have merge conflicts?

**Comment by reddraggone9** [comment]
I'm betting the code was written to *only* add labels.

Edit: Yep. [`label_pr`](https://github.com/chaosbot/chaos/blob/ef03e6cdbc97c385636213bc750d3da96db7c67c/github_api/prs.py#L84-L90) just adds a label. It's the only thing called [where the "mergeable" and "conflicts" labels are applied](https://github.com/chaosbot/chaos/blob/ef03e6cdbc97c385636213bc750d3da96db7c67c/github_api/prs.py#L144-L148).

**Comment by ECrownofFire** [comment]
https://developer.github.com/v3/pulls/#get-a-single-pull-request

>The value of the mergeable attribute can be true, false, or null. If the value is null, this means that the mergeability hasn't been computed yet, and a background job was started to compute it. Give the job a few moments to complete, and then submit the request again. When the job is complete, the response will include a non-null value for the mergeable attribute.

`get_is_mergeable` only checks for `True`. I think it'd be better to just transparently pass the value it gets.

**Comment by PlasmaPower** [comment]
@reddraggone9 #210

**Comment by PlasmaPower** [comment]
@ECrownofFire #212

Edit: you beat me to it :)

**Review by jblz — APPROVED** [review]

**Comment by yet-another-account** [comment]
Resolved conflicts

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@
 .venv
 *.secret
 *.swp
-.vagrant/
+.vagrant/
```
Comment (reddraggone9) [inline .gitignore]: What you got against line breaks?

Comment (yet-another-account) [reply .gitignore]: Must have accidentally destroyed it while resolving conflicts

**Pull request merged**